### DISCUSSION TITLE
Tags: let user choosing between even and odd numbers in the list or set custom value

### DISCUSSION THREAD

**Discussion by Sulpher** [joomla/joomla-cms#47043] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

I am using tags as a navigation through the content.
The problem occurs with pagination due to design: I want to display 3 items per row.
Example:
<img width="1289" alt="Снимок экрана 2024-05-20 в 09 55 29" src="https://github.com/joomla/joomla-cms/assets/3432048/46f9afa0-f0c9-4cfe-9464-745bc4e37883">

Menu item:
Compact List of Tagged Items
List Layouts tab > Items to list

<img width="866" alt="Снимок экрана 2024-05-20 в 09 52 12" src="https://github.com/joomla/joomla-cms/assets/3432048/acca3721-28ff-4eea-9443-4c4e525c863f">

Unfortunately, there is no way to choose odd numbers like 6, 9 or 12.

### Describe the solution you'd like

What about adding an extra input field to let a user write the right value? Then it is possible to set any desired value - items to list.

**Comment by Sulpher** [comment]
Maybe an extra field allowing to set custom value could help...

**Comment by denysdesign** [comment]
A good idea to expand the functionality, especially for customisation at the template level 👍

**Comment by alex-revo** [comment]
I agree with the author, I've encountered this problem myself.
